app/timetables/models.py

```diff
@@ -2,10 +2,12 @@
 
 from django.db import models
 
-from app.shared.models import CommonInfo
+from app.shared.models import TimestampMixin
 
 
-class Weekday(CommonInfo):
+class Weekday(TimestampMixin):
```

Comment:
If we would be using `TimestampMixin` for this class, then it should be class `Weekday(TimestampMixin, models.Model):`.